CRM/Utils/API/AbstractFieldCoder.php

```diff
@@ -63,6 +63,11 @@ public function isSkippedField($fldName) {
     if ($skipFields === NULL) {
       return FALSE;
     }
+    // Strip extra numbers from custom fields e.g. custom_32_1 should be custom_32
```

Comment:
@colemanw won't this possibly mean that all custom fields are then skipped not just rich text ones?

Reply:
No, it is just resolving custom fields to their canonical names which is `custom_{fid}`.
Quickform sometimes adds extra numbers to the end of the field name, e.g. `custom_{fid}_xx` so we're just stripping that extra stuff that isn't identifying the field.